ffi/athcon/bindings/rust/athcon-vm/src/lib.rs

```diff
@@ -53,6 +53,7 @@ pub struct ExecutionMessage {
   recipient: Address,
   sender: Address,
   input: Option<Vec<u8>>,
+  method: Option<Vec<u8>>,
```

Comment:
Why not use a String here? 
```suggestion
  method: Option<String>,
```

Reply:
https://github.com/athenavm/athena/issues/113